atomic

```diff
@@ -119,6 +116,15 @@ if __name__ == '__main__':
     parser.add_argument('--debug', default=False, action='store_true')
     subparser = parser.add_subparsers(help=_("commands"))
 
+    # atomic cluster (if available)
+    try:
+        import commctl.cli
+        clusterp = subparser.add_parser(
```

Comment:
I don't follow this, why should we require it? Atomic on its own is useful without commissaire (e.g. I like using atomic on my workstation, but I'm not planning to join a cluster anytime soon).